### PR TITLE
Add documentation for getbyshortid

### PR DESCRIPTION
**Pull request by jonathanjouty**
# Do not merge yet!

_To be merged after production update on 25th February_

New API Call in progress.
See Case 2624.

### CODE REVIEW

**Comment by jonathanjouty** [comment]
screen shot 2017-02-07 at 15 23 33
screen shot 2017-02-07 at 15 29 15

File: documentation/api/documents/short_document_id/getbyshortid.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  tags:
+    - Get
+
+  description: |
+    **Get the Document JSON metadata using a short Document ID.**
+
+    This can only be used for documents created or modified within the last 24
+    hours.
+    It can only be used for documents that are pending.
+    You can only get documents that you usually have access to.
```
Comment (mariuszrak) [inline documentation/api/documents/short_document_id/getbyshortid.yaml]: "usually have access"?

Comment (jonathanjouty) [inline documentation/api/documents/short_document_id/getbyshortid.yaml]: You are right, it sounds a bit odd :)
Should I just remove it completely? or reword?

Comment (mariuszrak) [inline documentation/api/documents/short_document_id/getbyshortid.yaml]: Just remove 'usually'

**Comment by jonathanjouty** [comment]
@mariuszrak fixed. Changed other wording a bit too...

**Pull request merged**